tutorials/spectral-cube-reprojection/SpectralCubeReprojectExample.ipynb

```diff
@@ -0,0 +1,640 @@
+{
```

Comment:
<p>Let's download some data of a BLAH object from the ALMA archives. These files are quite large (18 MB and 337 MB, respectively). </p><p> </p><p>See the <a href="https://docs.astropy.org/en/stable/api/astropy.utils.data.download_file.html" target="_blank">astropy utils documentation</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/504/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>modified</p>